src/InstallationValidator.Core/Presentation/Views/ISimulationComparisonView.cs

```diff
@@ -0,0 +1,14 @@
+using InstallationValidator.Core.Presentation.DTO;
+using OSPSuite.Presentation.Views;
+
+namespace InstallationValidator.Core.Presentation.Views
+{
+   public interface ISimulationComparisonView : IView<ISimulationComparisonPresenter>
+   {
+      void ResetText(string empty);
+      void AppendText(string textToLog);
+      void AppendHTML(string htmlToLog);
+      void ComparisonIsRunning(bool comparisonRunning);
+      void BindTo(FolderDTO firstFolderDTO, FolderDTO secondFolderDTO);
```

Comment:
Unusual signature here. We usually bind to one DTO that has all the info we need